helper/diffPlaces.js

```diff
@@ -169,11 +169,47 @@ function isAddressDifferent(item1, item2){
   return false;
 }
 
+function isGeonamesConcordanceSame(item1, item2) {
+  let wof_record;
+  let gn_record;
+
+  if (item1.source === 'geonames' && item2.source === 'whosonfirst') {
+    gn_record = item1;
+    wof_record = item2;
+  } else if (item2.source === 'geonames' && item1.source === 'whosonfirst') {
+    gn_record = item2;
+    wof_record = item1;
+  } else {
+    // could not match to one geonames and one wof concordance, so this check does not apply
+    return false;
+  }
+
+  const concordances = _.get(wof_record, 'addendum.concordances');
+
+  if (concordances) {
+    const json = JSON.parse(concordances);
```

Comment:
there is a 'codec' in `pelias/model` which is actually just using `JSON.parse` under-the-hood but serves as a nice abstraction for if we later want to change the encoding or apply compression or whatever.

there are some examples in this repo already, have a look for `const codec = require('pelias-model').codec;`